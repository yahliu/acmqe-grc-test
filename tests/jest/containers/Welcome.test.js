/* Copyright (c) 2021 Red Hat, Inc. */
/* Copyright Contributors to the Open Cluster Management project */

'use strict'

import React from 'react'
import renderer from 'react-test-renderer'
import WelcomeStatic from '../../../src-web/containers/Welcome'

describe('Welcome component', () => {
    it('renders as expected', () => {
      const component = renderer.create(
        <WelcomeStatic />
      )
      expect(component.toJSON()).toMatchSnapshot()
    })
})
