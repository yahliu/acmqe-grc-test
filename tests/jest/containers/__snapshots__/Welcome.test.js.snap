// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome component renders as expected 1`] = `
<div
  className="welcome"
>
  <div
    aria-label="Welcome to Red Hat Advanced Cluster Management for Kubernetes"
    className="welcome--introduction"
    role="region"
  >
    <div
      className="welcome--introduction__description"
    >
      <div
        className="welcome--text"
      >
        <h1>
          Welcome! Letâ€™s get started.
        </h1>
        <p>
          Red Hat Advanced Cluster Management for Kubernetes provides the tools and capabilities to address various challenges with managing multiple clusters and consoles, distributed business applications, and inconsistent security controls across Kubernetes clusters that are deployed on-premises, or across public clouds.
        </p>
      </div>
      <div
        className="welcome--image"
      >
        <img
          alt="Welcome image"
          height="32px"
          src="undefined/graphics/Welcome-page-header.png"
          width="32px"
        />
      </div>
    </div>
  </div>
  <div
    aria-label="Information on how to connect with Red Hat Advanced Cluster Management for Kubernetes"
    className="welcome--svcs"
    role="region"
  >
    <div
      className="welcome--svcs-row"
    >
      <div
        className="welcome--svcs-container"
      >
        <div
          className="welcome--svcs-container-img"
        >
          <img
            alt=""
            height="32px"
            src="undefined/graphics/welcome-card-1-icon.svg"
            width="32px"
          />
        </div>
        <div
          className="welcome--svcs-container-body"
        >
          <p
            className="title"
          >
            End-to-end visibility
          </p>
          <p
            className="desc"
          >
            View system alerts, critical application metrics, and overall system health. Search, identify, and resolve issues that are impacting distributed workloads using an operational dashboard designed for Site Reliability Engineers (SREs).
          </p>
          <div
            className="links"
          >
            <a
              className="links-link"
              href="/overview"
            >
              Go to Overview
            </a>
          </div>
        </div>
      </div>
      <div
        className="welcome--svcs-container"
      >
        <div
          className="welcome--svcs-container-img"
        >
          <img
            alt=""
            height="32px"
            src="undefined/graphics/welcome-card-2-icon.svg"
            width="32px"
          />
        </div>
        <div
          className="welcome--svcs-container-body"
        >
          <p
            className="title"
          >
            Cluster lifecycle
          </p>
          <p
            className="desc"
          >
            Create, update, scale, and remove clusters reliably, consistently using an open source programming model that supports and encourages Infrastructure as Code best practices and design principles.
          </p>
          <div
            className="links"
          >
            <a
              className="links-link"
              href="/multicloud/clusters"
            >
              Go to Clusters
            </a>
          </div>
        </div>
      </div>
    </div>
    <div
      className="welcome--svcs-row"
    >
      <div
        className="welcome--svcs-container"
      >
        <div
          className="welcome--svcs-container-img"
        >
          <img
            alt=""
            height="32px"
            src="undefined/graphics/welcome-card-3-icon.svg"
            width="32px"
          />
        </div>
        <div
          className="welcome--svcs-container-body"
        >
          <p
            className="title"
          >
            Application lifecycle
          </p>
          <p
            className="desc"
          >
            Define a business application using open standards and deploy the applications using placement policies that are integrated into existing CI/CD pipelines and governance controls.
          </p>
          <div
            className="links"
          >
            <a
              className="links-link"
              href="/multicloud/applications"
            >
              Go to Applications
            </a>
          </div>
        </div>
      </div>
      <div
        className="welcome--svcs-container"
      >
        <div
          className="welcome--svcs-container-img"
        >
          <img
            alt=""
            height="32px"
            src="undefined/graphics/welcome-card-4-icon.svg"
            width="32px"
          />
        </div>
        <div
          className="welcome--svcs-container-body"
        >
          <p
            className="title"
          >
            Governance, Risk, and Compliance
          </p>
          <p
            className="desc"
          >
            Use policies to automatically configure and maintain consistency of security controls required by industry or other corporate standards. Prevent unintentional or malicious configuration drift that might expose unwanted and unnecessary threat vectors.
          </p>
          <div
            className="links"
          >
            <a
              className="links-link"
              href="/multicloud/policies"
            >
              Go to Governance
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    aria-label="Information on how to connect with Red Hat Advanced Cluster Management for Kubernetes"
    className="welcome--info"
    role="region"
  >
    <div
      className="welcome--info__heading"
    >
      <h2>
        Easy, simple, and secure.
      </h2>
    </div>
    <div
      className="welcome--info__details"
    >
      <div
        className="welcome--info__details-top"
      >
        <p>
          Easy to use and simple to understand, Red Hat Advanced Cluster Management for Kubernetes provides the following mission critical capabilities based on open source projects:
        </p>
      </div>
      <div
        className="welcome--info__details-container"
      >
        <div
          className="container-row"
        >
          <div
            className="welcome--info__details-container-item spacer"
          >
            <p
              className="header"
            >
              Kubernetes
            </p>
            <p
              className="desc"
            >
              Easily provision Kubernetes clusters and offer complete cluster lifecycle management in a single console.
            </p>
          </div>
          <div
            className="welcome--info__details-container-item spacer"
          >
            <p
              className="header"
            >
              Policies
            </p>
            <p
              className="desc"
            >
              Enforce policies at the target clusters using Kubernetes-supported custom resource definitions.
            </p>
          </div>
          <div
            className="welcome--info__details-container-item"
          >
            <p
              className="header"
            >
              Cluster landscape
            </p>
            <p
              className="desc"
            >
              Deploy and maintain day two operations of applications distributed across your cluster landscape.
            </p>
          </div>
        </div>
        <div
          className="container-row"
        >
          <div
            className="welcome--info__details-container-item spacer"
          >
            <p
              className="header"
            >
              Range of environments
            </p>
            <p
              className="desc"
            >
              Work across a range of environments, including multiple data centers, and private and public clouds.
            </p>
          </div>
          <div
            className="welcome--info__details-container-item spacer"
          >
            <p
              className="header"
            >
              Application topology
            </p>
            <p
              className="desc"
            >
              Quickly view service endpoints, pods, and dependant resources that comprise your application topology.
            </p>
          </div>
          <div
            className="welcome--info__details-container-item"
          >
            <p
              className="header"
            >
              Cluster labels and placement rules
            </p>
            <p
              className="desc"
            >
              Use cluster labels and application placement rules to easily move workloads across clusters, even between multiple cloud providers.
            </p>
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    aria-label="Information on how to connect with Red Hat Advanced Cluster Management for Kubernetes"
    className="welcome--connect"
    role="region"
  >
    <div
      className="welcome--connect__heading"
    >
      <h2>
        Converse and connect.
      </h2>
    </div>
    <div
      className="welcome--connect__details"
    >
      <div
        className="welcome--connect__section"
      >
        <img
          alt="Video blog icon"
          className="welcome--img"
          src="undefined/graphics/welcome-community-icon.svg"
        />
        <p
          className="welcome--connect__section-title"
        >
          Technical community
        </p>
        <p
          className="welcome--connect__section-body"
        >
          Want more information? Check out the developer community.
        </p>
        <a
          aria-describedby="launchWindow"
          className="welcome--link"
          href="https://www.redhat.com/en/blog/products"
          target="dev-community"
        >
          Learn more
          <img
            alt="Learn more"
            className="welcome--connect__section-launch"
            src="undefined/graphics/arrow-right.svg"
          />
        </a>
      </div>
      <div
        className="welcome--connect__section"
      >
        <img
          alt="Slack icon"
          className="welcome--img"
          src="undefined/graphics/welcome-slack-icon.svg"
        />
        <p
          className="welcome--connect__section-title"
        >
          Slack channel
        </p>
        <p
          className="welcome--connect__section-body"
        >
          Join our Slack community to connect and chat with us.
        </p>
        <p
          className="connect-coming-soon"
        >
          Coming soon
        </p>
      </div>
      <div
        className="welcome--connect__section"
      >
        <img
          alt="Question mark icon"
          className="welcome--img"
          src="undefined/graphics/welcome-support-icon.svg"
        />
        <p
          className="welcome--connect__section-title"
        >
          Support center
        </p>
        <p
          className="welcome--connect__section-body"
        >
          Need technical support? Contact us.
        </p>
        <a
          aria-describedby="launchWindow"
          className="welcome--link"
          href="https://access.redhat.com/support"
          target="support"
        >
          Learn more
          <img
            alt="Learn more"
            className="welcome--connect__section-launch"
            src="undefined/graphics/arrow-right.svg"
          />
        </a>
      </div>
    </div>
  </div>
</div>
`;
